src/zui/ZUIOrganizeSidebar.tsx

```diff
@@ -1,219 +1,222 @@
+import ChevronLeftIcon from '@mui/icons-material/ChevronLeft';
+import CssBaseline from '@mui/material/CssBaseline';
+import KeyboardDoubleArrowRightIcon from '@mui/icons-material/KeyboardDoubleArrowRight';
```

Comment:
Instead of importing the icon like this, just `import { KeyboardDoubleArrowRight } from '@mui/icons-material`.

Reply:
Of course 😓